qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/DependencyManager.groovy

```diff
@@ -77,11 +83,28 @@ class DependencyManager {
         return this.portletView
     }
 
+    private void setupDbConnections() {
+        try {
+
+            String user = configurationManager.getMysqlUser()
+            String password = configurationManager.getMysqlPass()
+            String host = configurationManager.getMysqlHost()
+            String port = configurationManager.getMysqlPort()
+            String sqlDatabase = configurationManager.getMysqlDB()
```

Comment:
What happens if the property is not set? Maybe wrap a `Objects.requireNotNull()` around it.

Reply:
will do

Reply:
45476bc